qiskit/circuit/library/standard_gates/z.py

```diff
@@ -127,7 +127,15 @@ def control(
         return gate
 
     def inverse(self, annotated: bool = False):
-        """Return inverted Z gate (itself)."""
+        """Return inverted Z gate (itself).
+
+        Args:
+            annotated: indicates whether the inverse gate can be implemented
+                as an annotated gate.
```

Comment:
```suggestion
            annotated: Typically when set to ``True`` this is used to return an :class:`.AnnotatedOperation` 
                with an inverse modifier set instead of a concrete :class:`.Gate`. However, for this class this
                argument is ignored as this gate is self inverse.
```